smart-contracts/contracts/utils/UnlockOwner.sol

```diff
@@ -0,0 +1,226 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.7;
+
+/**
+ * UnlockManager is used to facilitate the governance of the Unlock protocol
+ * accross multiple chains. His role is to manage the state of the Unlock contract
+ * based on instructions coming from the DAO on mainnet via the Connext bridge.
+ * 
+ * There are two kind of instructions that can be sent :
+ * - 1. change in Unlock settings
+ * - 2. upgrade of the Unlock contract (via its proxyAdmin)
+ *
+ * As a security measure, this contract also has an `exec` function 
+ * that can be used to send calls directly (without going through a bridge)
+ * through a multisig wallet managed by the Unlock team.
+ * 
+ */
+
+
+import {IXReceiver} from "@connext/nxtp-contracts/contracts/core/connext/interfaces/IXReceiver.sol";
+import {IConnext} from "@connext/nxtp-contracts/contracts/core/connext/interfaces/IConnext.sol";
+import '../interfaces/IUnlock.sol';
+
+contract UnlockOwner {
+
+  // address of the connext bridge on the current chain
+  address public bridgeAddress;
+
+  // address of Unlock on the current chain
+  address public unlockAddress;
+
+  // address of Unlock team multisig wallet for the current chain
+  address public multisigAddress;
+  
+  // address of the DAO on mainnet (used to verify trusted origin)
+  address public daoAddress;
```

Comment:
In fact this needs to be the Timelock address because that is from there that the transactions will be sent. For clarity lets use: 

```suggestion
  address public daoTimelockAddress;
```